profile/README.md

```diff
@@ -1,30 +1,23 @@
-  * [RoiExtractors-GPL](https://github.com/catalystneuro/roiextractors-gpl): An extension to RoiExtractors including imports of GPL licenses
+## About Us 🧑‍💻
+[CatalystNeuro](http://catalystneuro.com) is a software consulting company that specializes in data engineering for neuroscience labs. We are a globally distributed team of n neuroscientists who are also software developers, and we care deeply about helping neuroscience adopt state-of-the-art data engineering tools. We excel at helping data infrastructure tools with outreach and with helping neuroscience labs adopt data infrastructure tools. See our team [here](https://www.catalystneuro.com/team/).
 
-### Lab-specific Neurodata Without Borders (NWB) conversion projects
-* [Movshon Lab, NYU](https://github.com/catalystneuro/movshon-lab-to-nwb): Blackrock recordings (under development)
-* [Brunton Lab, UW](https://github.com/catalystneuro/brunton-lab-to-nwb): ECoG with natural arm movement (dandiset [#55](https://dandiarchive.org/dandiset/000055/draft))
-* [Giocomo Lab, Stanford](https://github.com/catalystneuro/giocomo-lab-to-nwb): SpikeGLX Neuropixel recordings on virtual linear track (dandiset [#53](https://dandiarchive.org/dandiset/000053/draft))
-* [Tank Lab, Princeton](https://github.com/catalystneuro/tank-lab-to-nwb): SpikeGLX Neuropixel recordings in a decision making task
-  * [ndx-tank-metadata](https://github.com/catalystneuro/ndx-tank-metadata): stores information about the [ViRMEn MATLAB VR program](https://pni.princeton.edu/pni-software-tools/virmen)
-* [Allen Institute](https://github.com/catalystneuro/allen-oephys-to-nwb): Simultaneous optical imaging and electrophysiology
-* [Buzsaki Lab, NYU](https://github.com/catalystneuro/buzsaki-lab-to-nwb): NeuroScope recordings of various navigation experiments (dandiset [#3](https://dandiarchive.org/dandiset/000003/draft))
-* [Groh Lab, Heidelberg](https://github.com/catalystneuro/mease-lab-to-nwb): CED and Syntalos recording systems
-* [Buffalo Lab, UW](https://github.com/catalystneuro/buffalo-lab-to-nwb): Blackrock recordings of NHPs in virtual environments
-* [Jaeger Lab, Emory](https://github.com/catalystneuro/jaeger-lab-to-nwb): Intan electrophysiology and FRET optical imaging
-  * [ndx-FRET](https://github.com/catalystneuro/ndx-fret)
+### Key Software Projects 📈
+* [NeuroConv](https://github.com/catalystneuro/neuroconv) automates conversion of data from common formats into the Neurodata Without Borders (NWB) format for distribution on the DANDI Archive.
+* [NWBWidgets](https://github.com/NeurodataWithoutBorders/nwbwidgets) provides automatic, interactive visualizations of the data within NWB files that are scalable and integrated with the DANDI Archive.
+* [SpikeInterface](https://github.com/SpikeInterface/spikeinterface) is a community-driven unified framework for spike sorting.
```

Comment:
Since this README is logo-heavy, want to add in the logos for NeuroConv and SI?

(Widgets could probably use one too one of these days...)

Reply:
I'd rather not do that unless we have logos for all of them. I can't find the logo for NeuroConv. Do you have the file handy?